tests/job/test_jobs.py

```diff
@@ -54,39 +56,38 @@ def test_succeeded(self):
         self.check_resp_succeeded(resp)
         expect_col_names = ['Job Id', 'Command', 'Status', 'Start Time', 'Stop Time']
         self.check_column_names(resp, expect_col_names)
-        expect_values = [[re.compile(r'\d+'), re.compile(r'COMPACT'), re.compile(r'\S+'), re.compile(r'\d+'), re.compile(r'\d+')],
-                         [re.compile(r'\d+'), re.compile(r'FLUSH'), re.compile(r'\S+'), re.compile(r'\d+'), re.compile(r'\d+')],
-                         [re.compile(r'\d+'), re.compile(r'STATS'), re.compile(r'\S+'), re.compile(r'\d+'), re.compile(r'\d+')]]
-        self.search_result(resp, expect_values, is_regex=True)
+        assert resp.row_values(0)[1].as_string() ==  'STATS'
+        assert resp.row_values(0)[2].is_string()
+        assert resp.row_values(0)[3].is_datetime()
+        assert resp.row_values(0)[4].is_datetime()
```

Comment:
extract the common logics to a general function, such as: `check_job(job)`

Reply:
Fixed.